perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -40,23 +40,23 @@ then
 fi
 
 run_load_test_and_fetch_metrics(){
-  GCSFUSE_FIO_FLAGS=$1
-  UPLOAD_FLAGS=$2
-  BUCKET_NAME=$3
-  SPREADSHEET_ID=$4
-  GSHEET_CREDENTIALS=$5
-  TEST_TYPE=$6
-  GCSFUSE_LS_FLAGS=$7
-  LS_CONFIG_FILE=$8
+  gcsfuse_fio_flags=$1
```

Comment:
Ideally we should catch gcsfuse_flags only and then create fio_flag and ls_flags in the function.

Reply:
Updated.